pkg/resource/training_job/testdata/sdkapi/describe/warmpool_describe_available.json

```diff
@@ -0,0 +1,191 @@
+            },
+            {
+                "Name": "validation:map",
+                "Regex": ".*\\[[0-9]+\\].*#011validation-map:([-+]?[0-9]*\\.?[0-9]+(?:[eE][-+]?[0-9]+)?).*"
+            }
+        ],
+        "TrainingImage": "246618743249.dkr.ecr.us-west-2.amazonaws.com/xgboost:1",
+        "TrainingInputMode": "File"
+    },
+    "AutoMLJobArn": null,
+    "BillableTimeInSeconds": null,
+    "CheckpointConfig": null,
+    "CreationTime": "2021-10-12T05:49:40.493Z",
+    "DebugHookConfig": null,
+    "DebugRuleConfigurations": null,
+    "DebugRuleEvaluationStatuses": null,
+    "EnableInterContainerTrafficEncryption": false,
+    "EnableManagedSpotTraining": false,
+    "EnableNetworkIsolation": false,
+    "Environment": null,
+    "ExperimentConfig": null,
+    "FailureReason": null,
+    "FinalMetricDataList": null,
+    "HyperParameters": {
+        "eta": "0.2",
+        "gamma": "4",
+        "max_depth": "5",
+        "min_child_weight": "6",
+        "num_class": "10",
+        "num_round": "10",
+        "objective": "multi:softmax",
+        "silent": "0"
+    },
+    "InputDataConfig": [
+        {
+            "ChannelName": "train",
+            "CompressionType": "None",
+            "ContentType": "text/csv",
+            "DataSource": {
+                "FileSystemDataSource": null,
+                "S3DataSource": {
+                    "AttributeNames": null,
+                    "S3DataDistributionType": "FullyReplicated",
+                    "S3DataType": "S3Prefix",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/train"
+                }
+            },
+            "InputMode": null,
+            "RecordWrapperType": "None",
+            "ShuffleConfig": null
+        },
+        {
+            "ChannelName": "validation",
+            "CompressionType": "None",
+            "ContentType": "text/csv",
+            "DataSource": {
+                "FileSystemDataSource": null,
+                "S3DataSource": {
+                    "AttributeNames": null,
+                    "S3DataDistributionType": "FullyReplicated",
+                    "S3DataType": "S3Prefix",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/validation"
+                }
+            },
+            "InputMode": null,
+            "RecordWrapperType": "None",
+            "ShuffleConfig": null
+        }
+    ],
+    "LabelingJobArn": null,
+    "LastModifiedTime": "2021-10-12T05:52:46.108Z",
+    "ModelArtifacts": null,
+    "OutputDataConfig": {
+        "KmsKeyId": "",
+        "S3OutputPath": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/training/output"
+    },
+    "ProfilerConfig": null,
+    "ProfilerRuleConfigurations": null,
+    "ProfilerRuleEvaluationStatuses": null,
+    "ProfilingStatus": "Disabled",
+    "ResourceConfig": {
+        "InstanceCount": 1,
```

Comment:
shouldnt there be a keep alive time here?

Reply:
Keep alive time only appears in api responses after the warmpool cluster gets terminated or gets reused. I've noticed it with boto3/ack.